data/state/account.go

```diff
@@ -233,7 +233,7 @@ func (a *Account) AddToBalance(value *big.Int) error {
 	err := a.SetBalanceWithJournal(newBalance)
 	if err != nil {
 		log.Debug("SetDeveloperRewardWithJournal error", "error", err.Error())
```

Comment:
Do you still need to log it here if the error is returned?

Reply:
deleted